### PR TITLE
add parakeet finetuning tutorials

### PR DESCRIPTION
**Pull request by jmayank1511**

### CODE REVIEW

**Comment by rmittal-github** [comment]
rebased and added a minor fix found in other thread, @nv-uvaidya could you review the new tutorial as well?

File: asr_finetune_parakeet_nemo.ipynb [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+      "metadata": {
+        "id": "4NpvEOMs_mKp"
+      },
+      "source": [
+        "<img src=\"http://developer.download.nvidia.com/notebooks/dlsw-notebooks/rivaasrasr-finetuning-conformer-ctc-nemo/nvidia_logo.png\" style=\"width: 90px; float: right;\">\n",
+        "\n",
+        "# How to Fine-Tune a Riva ASR Acoustic Model with NVIDIA NeMo\n",
+        "This tutorial walks you through how to fine-tune an NVIDIA Riva ASR acoustic model with NVIDIA NeMo.\n",
+        "\n",
+        "**Important**: If you plan to fine-tune an ASR acoustic model using the same tokenizer with which the model was trained, skip this tutorial and refer to the \"Sub-word Encoding CTC Model\" section (starting with the \"Load pre-trained model\" subsection) of the [NeMo ASR Language Finetuning tutorial](https://github.com/NVIDIA/NeMo/blob/main/tutorials/asr/ASR_CTC_Language_Finetuning.ipynb)."
```
[the remark this answers is not in the document]
Comment (myungjongk) [reply asr_finetune_parakeet_nemo.ipynb]: @jmayank1511

File: asr_finetune_parakeet_nemo.ipynb [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "6-yDMTMO_mMJ"
+      },
+      "source": [
+        "#### Training Parakeet-Hybrid model\n",
+        "\n",
+        "Hybrid RNNT-CTC models is a group of models with both the RNNT and CTC decoders. Training a hybrid model would speedup the convergence for the CTC models and would enable the user to use a single model which works as both a CTC and RNNT model. This category can be used with any of the ASR models. Hybrid models uses two decoders of CTC and RNNT on the top of the encoder.\n",
+        "\n",
+        "NeMo uses `.yml` files to configure the training parameters. You may update them directly by editing the configuration file or from the command-line interface. For example, if the number of epochs needs to be modified, along with a change in the learning rate, you can add `trainer.max_epochs=100` and `optim.lr=0.02` and train the model.\n",
```
[the remark this answers is not in the document]
Comment (myungjongk) [reply asr_finetune_parakeet_nemo.ipynb]: @jmayank1511

File: asr_finetune_parakeet_nemo.ipynb [hunk 3]
```diff
@@ -0,0 +1,506 @@
...
+        "\n",
+        "The following sample command uses the `speech_to_text_hybrid_rnnt_ctc_bpe.py` script in the `examples` folder to train/fine-tune a Parakeet-Hybrid ASR model for 1 epoch. For other ASR models like Citrinet, Conformer, you may find the appropriate config files in the NeMo GitHub repo under [examples/asr/conf/](https://github.com/NVIDIA/NeMo/tree/main/examples/asr/conf).\n"
```
[the remark this answers is not in the document]
Comment (myungjongk) [reply asr_finetune_parakeet_nemo.ipynb]: @jmayank1511

File: asr_finetune_parakeet_nemo.ipynb [hunk 4]
```diff
@@ -0,0 +1,506 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8XNolXIPptAr"
+      },
+      "source": [
+        "#### Convert to Riva\n",
+        "\n",
+        "Convert the downloaded model to the `.riva` format. We will set the encryption key with `--key=nemotoriva`. Choose a different encryption key value when generating `.riva` models for production.\n",
```
[the remark this answers is not in the document]
Comment (myungjongk) [reply asr_finetune_parakeet_nemo.ipynb]: @jmayank1511

File: asr_finetune_parakeet_nemo.ipynb [hunk 5]
```diff
@@ -0,0 +1,506 @@
...
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "PlUNmVHKp1ft"
+      },
+      "outputs": [],
+      "source": [
+        "riva_file_path = ctc_model_path[:-5]+\".riva\"\n",
+        "!nemo2riva --key=nemotoriva  --out $riva_file_path $ctc_model_path"
```
[the remark this answers is not in the document]
Comment (myungjongk) [reply asr_finetune_parakeet_nemo.ipynb]: @jmayank1511

File: asr_finetune_parakeet_nemo.ipynb (hunk 4 above)
[the remark this answers is not in the document]
Comment (jmayank1511) [reply asr_finetune_parakeet_nemo.ipynb]: not needed for RNNT models

**Review by myungjongk — APPROVED** [review]
Looks good!

**Review by rmittal-github — APPROVED** [review]

**Pull request merged**